v2/Brewfile

```diff
@@ -0,0 +1,19 @@
+# Modern bash
+brew "bash"
+
+# Command line completion
+brew "bash-completion"
+brew "zsh-completions"
+brew "docker-completion"
```

Comment:
would it make more sense if this line came after the below line:

brew "docker-compose"

?

Reply:
it does make sense to put this block (of completions) at the very end